src/manage-fonts/confirm-delete-modal.js

```diff
@@ -0,0 +1,50 @@
+import {
+	// eslint-disable-next-line
+	__experimentalConfirmDialog as ConfirmDialog,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+
+function ConfirmDeleteModal( { isOpen, onConfirm, onCancel, fontToDelete } ) {
+	return (
+		<ConfirmDialog
+			isOpen={ isOpen }
+			onConfirm={ onConfirm }
+			onCancel={ onCancel }
+		>
+			{ fontToDelete?.weight !== undefined &&
+			fontToDelete.style !== undefined ? (
+				<h3>
+					{ sprintf(
```

Comment:
move the messages to variables, and use them here?

Reply:
> I am unable to test directly on this branch. It doesn't render anything. (after running build) Also having hard time resolving the conflicts if I try to rebase.
> 
> Can you rebase the changes with trunk?

Is it possible that your local env is corrupted in some way?
The branch is up-to-date and merging with latest trunk adds no change or conflict for me.